test/e2e/util.go

```diff
@@ -420,29 +420,29 @@ func TearDown(namespace string) {
 		return client.DeleteAllOf(clientCtx, &operatorsv1alpha1.InstallPlan{}, inNamespace)
 	}).Should(Succeed(), "failed to delete test installplans")
 
-	Eventually(func() (remaining []operatorsv1alpha1.InstallPlan, err error) {
-		list := &operatorsv1alpha1.InstallPlanList{}
-		err = client.List(clientCtx, list, inNamespace)
-		if list != nil {
-			remaining = list.Items
+	var installPlangvr = schema.GroupVersionResource{Group: "operators.coreos.com", Version: "v1alpha1", Resource: "installplans"}
+	Eventually(func() ([]unstructured.Unstructured, error) {
+	fmt.Printf("checking installplan\n")
+		list, err := dynamic.Resource(installPlangvr).Namespace(namespace).List(context.Background(), metav1.ListOptions{})
+		if err != nil {
+			return nil, err
 		}
-
-		return
+		return list.Items, nil
 	}).Should(BeEmpty(), "failed to await deletion of test installplans")
 
 	logf("deleting test catalogsources...")
 	Eventually(func() error {
 		return client.DeleteAllOf(clientCtx, &operatorsv1alpha1.CatalogSource{}, inNamespace, ephemeralCatalogFieldSelector)
 	}).Should(Succeed(), "failed to delete test catalogsources")
 
-	Eventually(func() (remaining []operatorsv1alpha1.CatalogSource, err error) {
-		list := &operatorsv1alpha1.CatalogSourceList{}
-		err = client.List(clientCtx, list, inNamespace, ephemeralCatalogFieldSelector)
-		if list != nil {
-			remaining = list.Items
+	var catalogSourcegvr = schema.GroupVersionResource{Group: "operators.coreos.com", Version: "v1alpha1", Resource: "catalogsources"}
+	Eventually(func() ([]unstructured.Unstructured, error) {
+	fmt.Printf("checking catalogsource\n")
```

Comment:
Using `fmt` here as well, can be closed / changed based on your reasoning [here](https://github.com/operator-framework/operator-lifecycle-manager/pull/2676/files#r821794321).

Reply:
Sorry, it was for my debug.  I'll take out.